releasenotes/notes/add-anthropic-streaming-support-01937d2e524f1bd0.yaml

```diff
@@ -1,4 +1,5 @@
 ---
 features:
   - |
-    Anthropic: Add message streaming and async messaging streaming support.
+    Anthropic: Adds support for tracing synchronous and asynchronous message streaming.
+    LLM Observability: Adds support for tracing synchronous and asynchronous message streaming.
```

Comment:
are these meant to be two separate notes?
```suggestion
    Anthropic: Adds support for tracing synchronous and asynchronous message streaming.
  - |
    LLM Observability: Adds support for tracing synchronous and asynchronous message streaming.
```

?

Reply:
would it be better just as:

```suggestion
    LLM Observability: Adds support for tracing synchronous and asynchronous message streaming with Anthropic.
```

(no opinion, just a different approach I thought of while commenting)

Reply:
These are two separate release features as this adds streaming support for apm traces as well as llmobs - I'd prefer to keep them two release notes as having them just be under `LLM Observability` leaves it ambiguous whether or not we support streaming for regular APM spans

Reply:
Meant to be two seperate notes, resolving as per offline discussion with @Yun-Kim

Reply:
ok, then we'll want the first suggested change.

as-is, they are the same note, you want to have two list items:

```yaml
- |
  First note 
- |
  Second note
```